README.md

```diff
@@ -111,37 +111,16 @@ Install this project with:
 
 To run the project, execute the following command:
 
-<!--
-Example command:
-
-```sh
-  rails server
-```
---->
 
 ### Run tests
 
 To run tests, run the following command:
 
-<!--
-Example command:
-
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
 
 ### Deployment
 
-You can deploy this project using:
-
-<!--
-Example:
-
-```sh
+You can deploy this project using: https://joshualyngdoh.github.io/Portfolio/
```

Comment:
- [ ] [OPTIONAL] Good job deploying your project 🥇 🥇 However, I think it would be better to put your deployment link below the **Live demo** title to make it easily found by the user 👍 

_you may kindly add it like this 👇_ 
``` [Live Demo Link](https://joshualyngdoh.github.io/Portfolio)```

Reply:
Okay. Thank you for the feedback.